articles-1.html

```diff
@@ -11,17 +11,17 @@
 			<nav name="top">
 				<ul>
 					<li><a href="quiz.html">skin type quiz</a></li>
-					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+					<li class="dropdown"><a href="products-1.html">products &#9662;</a>

```

Comment:
In version 3, I changed the file name to "articles-1" as I thought it was more relevant since it doesn't have the entire article list on one page as the old file name implied.